native-script.rb

```diff
@@ -0,0 +1,42 @@
+# coding: utf-8
+
+puts "Installing Homebrew (a sudo password prompt may appear)."
+`ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+
+puts "Installing Homebrew-Cask (a sudo password prompt may appear)."
+`brew install caskroom/cask/brew-cask`
+
+puts "Installing Java (a sudo password prompt may appear)."
```

Comment:
Installing Java...
(You might need to provide your password.)

Btw, are we installing java or jdk? (Because we're actually requiring JDK)